docs/install-ilab-with-wsl.md

```diff
@@ -0,0 +1,62 @@
+# Installing InstructLab on Windows using WSL (Windows Subsystem for Linux)
+
```

Comment:
Can we put a big "NOT SUPPORTED" or something on this document? Opening up any Windows support should be something we discuss before putting this in any official documentation.